### PR TITLE
[ADPRODUCTS-367] fill NAs with False

### PR DESCRIPTION
**Pull request by rebecca-burwei**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_suggest_v1/query.sql [hunk 1]
```diff
@@ -7,14 +7,16 @@ WITH combined AS (
     LOWER(advertiser) AS advertiser,
     SPLIT(metadata.user_agent.os, ' ')[SAFE_OFFSET(0)] AS normalized_os,
     release_channel,
-    position,
+    -- 1-based position
+    (position + 1) AS position,
```
Comment (curtismorales) [inline sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_suggest_v1/query.sql:11]:
```suggestion
    position,
```

File: sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_suggest_v1/query.sql [hunk 2]
```diff
@@ -7,14 +7,16 @@ WITH combined AS (
...
     IF(request_id IS NULL, 'remote settings', 'merino') AS provider,
     match_type,
-    (
+    coalesce(
```
Comment (curtismorales) [inline sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_suggest_v1/query.sql:14]:
```suggestion
    COALESCE(
```

File: sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_suggest_v1/query.sql [hunk 3]
```diff
@@ -28,14 +30,16 @@ WITH combined AS (
     LOWER(advertiser) AS advertiser,
     SPLIT(metadata.user_agent.os, ' ')[SAFE_OFFSET(0)] AS normalized_os,
     release_channel,
+    -- 1-based position
     position,
     IF(request_id IS NULL, 'remote settings', 'merino') AS provider,
     match_type,
-    (
+    coalesce(
```
Comment (curtismorales) [inline sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_suggest_v1/query.sql:37]:
```suggestion
    COALESCE(
```

**Comment by rebecca-burwei** [comment]
I implemented a temporary fix for the strange values in position. But Eng is still investigating. I'd like to give them a day or two to see if they come up with anything.

**Review by curtismorales — APPROVED** [review]

**Pull request merged**